typescript/ecs/fargate-service-with-logging/index.ts

```diff
@@ -10,14 +10,12 @@ class WillkommenFargate extends cdk.Stack {
     const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
 
     // create a task definition with CloudWatch Logs
-    const logging = new ecs.AwsLogDriver(this, "AppLogging", {
+    const logging = new ecs.AwsLogDriver({
       streamPrefix: "myapp",
     })
 
-    const taskDef = new ecs.FargateTaskDefinition(this, "MyTaskDefinition", {
-      memoryMiB: '512',
```

Comment:
This should be `memoryLimitMiB: 512`. Removing the quotes around 512 will fix the issue.

Reply:
@pkandasamy91 according to `FaragateTaskDefinition` the default `memoryLimitMib` is 512 which is why I removed it.

Reply:
Ok, I'll bring them back up